src/app/review-item/services/reviews/reviews.service.ts

```diff
@@ -1,27 +1,39 @@
 import { Injectable } from '@angular/core';
 import { Review } from '../../model/review';
 import { API } from 'aws-amplify';
+import { from, Observable } from 'rxjs';
 
 @Injectable()
 export class ReviewsService implements IReviewService {
+  private apiName = 'review_service';
   private reviewsUrl = '/reviews';
 
   constructor() {}
 
+  getOpenReview(): Observable<Review> {
```

Comment:
I would like to have similar method signatures at least across one service.
For me it's not important if the methods return a Observable or a Promise, but I think it should be identical in one class.